fastlane/Fastfile

```diff
@@ -136,7 +136,31 @@ platform :android do
     # restrictions we have in `main`, and the release managers wouldn't be able to push due to permissions.
     # This should be changed only when we have PC Android releases done on CI, when the CI bot is the one running `git push`.
     set_branch_protection(repository: GH_REPOSITORY, branch: "release/#{new_version}")
-    set_milestone_frozen_marker(repository: GH_REPOSITORY, milestone: new_version)
+
+    begin
+      # Move PRs to next milestone
+      moved_prs = update_assigned_milestone(
+        repository: GH_REPOSITORY,
+        from_milestone: new_version,
+        to_milestone: release_version_next,
+        comment: "Version `#{new_version}` has now entered code-freeze, so the milestone of this PR has been updated to `#{release_version_next}`."
+      )
+      UI.message("Moved the following PRs to milestone #{release_version_next}: #{moved_prs.join(', ')}")
+
+      # Add ❄️ marker to milestone title to indicate we entered code-freeze
+      set_milestone_frozen_marker(
+        repository: GH_REPOSITORY,
+        milestone: new_version
+      )
+    rescue StandardError => e
+      moved_prs = []
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-android/blob/14b7e20c8217e600dd05302a97a4056aa980069a/fastlane/Fastfile#L156"></span>Lint/UselessAssignment: Useless assignment to variable - <code>moved_prs</code>.</td>
    </tr>
  </tbody>
</table>